crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -396,18 +396,26 @@ spec:
         - description: "Find: slow edges (> 1s)"
           expression: "rt > 1000"
         - description: "Find: unhealthy nodes"
-          expression:  "! healthy"
+          expression: "! healthy"
         - description: "Find: unknown nodes"
-          expression:  "name = unknown"
+          expression: "name = unknown"
         hide_options:
         - description: "Hide: healthy nodes"
           expression: "healthy"
         - description: "Hide: unknown nodes"
-          expression:  "name = unknown"
+          expression: "name = unknown"
         traffic:
           grpc: "requests"
           http: "requests"
-          tcp:  "sent"
+          tcp: "sent"
+      i18n:
+        language: "en"
+        show_selector: true
```

Comment:
I think this should be false.  We don't want to show the selector by default, most users will just want English.